client/app/bundles/shared/components/Checkbox.jsx

```diff
@@ -0,0 +1,46 @@
+// @flow
+import React from "react";
+import css from "./Checkbox.scss";
+
+type Props = {
+  id: string,
+  value?: string | number,
+};
+
+type State = {
+  checkboxState: boolean;
+};
+
+export default class Checkbox extends React.Component<Props, State>{
+  constructor(props: Props) {
+    super(props);
+    this.state = { checkboxState: true};
+  }
+
+  toggle = () => {
+    this.setState({ checkboxState: !this.state.checkboxState });
+  }
+
+  render() {
+    const checkedOrNot = [];
+    checkedOrNot.push(
+      <p>{this.state.checkboxState ? 'Unchecked' : 'Checked'}</p>
+    );
+    return (
+      <span>
+        <form>
+            <label className={css.switch}>Option1
+              <input type="checkbox" tabIndex="1" onChange={this.toggle}/>
+                <div className={css.check}></div>
+            </label>
+            <br/>
```

Comment:
Instead of line breaks, we should explicitly set a px value for the spacing between these two labels.